frontend/src/Commons/Forms/InputModules/FieldsAndSoil/FieldsAndSoil.tsx

```diff
@@ -180,16 +210,64 @@ const FieldsAndSoilComponent: React.FC<InputModuleProps> = ({
                     component="div"
                     className="errorMessage"
                   />
+                  {values.hasSoilTest === false && (
+                    <StyledWarningBlock>
+                      <p>
+                        For fields without a soil test, very high soil P and K fertility and a pH of
+                        4.0 will be assumed.
+                      </p>
+                    </StyledWarningBlock>
+                  )}
+                  {values.hasSoilTest && (
+                    <>
+                      <StyledSelectContainer>
+                        <CustomSelect
+                          name="SoilTest.TestingMethod"
+                          id="SoilTest.TestingMethod"
+                          label="Lab (Soil Test Methods)"
+                          options={soilTestOptions}
+                        />
+                      </StyledSelectContainer>
+                      <HeaderLabel>Soil Test values (top 6 inches of soil)</HeaderLabel>
```

Comment:
> [!WARNING]
> There's a few instances of this, but this is one of the bigger ones. 
`HeaderLabel` is a div, it's plain text wrapped in a div. 
This isn't great for Accessibility, but you could wrap the text in an appropriate header here. Since you use it in a few places I wouldnt lock the div into a specifically sized header, You can keep it as a div. 
```ts
<HeaderLabel>
   <h3>
      Text here
   </h3>
</HeaderLabel>
```
> [!NOTE]
> I arbitrarily picked a headersize. I didn't check what would be appropriate given your current hierarchy, Bear this in mind

![image](https://github.com/bcgov/nr-sustainment-capstone-2024/assets/62873746/90ec7b77-1c72-4848-845a-f832e783d11a)

It's also the same size as the rest of your text, so having that separation here can make a good difference too for people that need this technology